routes/handler.js

```diff
@@ -137,7 +140,8 @@ router.post("/register", function (req, res) {
             from: 'Perception 2020 Team, CETB',
             to: user.username,
             subject: 'Your password has changed',
```

Comment:
Change the subject to "Perception 2020 | Password reset"

Reply:
Understood